manifests/storage_migration_crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: storageversionmigrations.migration.k8s.io
```

Comment:
api-approval annotation is required when creating v1 CRDs using k8s.io suffix

Reply:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2337-k8s.io-group-protection/README.md

Reply:
https://github.com/kubernetes/community/pull/2524 looks like where @lavalamp acked the API